alm/src/config/descriptions.ts

```diff
@@ -9,29 +9,58 @@ export const TRANSACTION_STATUS_DESCRIPTION: { [key: string]: string } = {
 }
 
 export const CONFIRMATIONS_STATUS_LABEL: { [key: string]: string } = {
+  SUCCESS: 'Success',
+  SUCCESS_MESSAGE_FAILED: 'Success',
+  FAILED: 'Failed',
+  PENDING: 'Pending',
+  WAITING_VALIDATORS: 'Waiting',
+  WAITING_CHAIN: 'Waiting'
+}
+
+export const CONFIRMATIONS_STATUS_LABEL_HOME: { [key: string]: string } = {
   SUCCESS: 'Success',
   SUCCESS_MESSAGE_FAILED: 'Success',
   EXECUTION_FAILED: 'Execution failed',
   EXECUTION_PENDING: 'Execution pending',
   EXECUTION_WAITING: 'Execution waiting',
-  FAILED: 'Failed',
-  PENDING: 'Pending',
-  WAITING: 'Waiting'
+  FAILED: 'Confirmation Failed',
+  PENDING: 'Confirmation Pending',
+  WAITING_VALIDATORS: 'Confirmation Waiting',
+  WAITING_CHAIN: 'Confirmation Waiting'
 }
 
-// %homeChain will be replaced by the home network name
-// %foreignChain will be replaced by the foreign network name
+// use %link to identify a link
 export const CONFIRMATIONS_STATUS_DESCRIPTION: { [key: string]: string } = {
   SUCCESS: '',
   SUCCESS_MESSAGE_FAILED:
```

Comment:
The scenario `SUCCESS_MESSAGE_FAILED` was not covered in the descriptions provided. Please check the text I added for this case